mailbag/formats/msg.py

```diff
@@ -0,0 +1,36 @@
+from mailbag.email_account import EmailAccount
+from mailbag.models import Email
+import extract_msg
+import glob
+
+
+class MSG(EmailAccount):
+    """MSG - This concrete class parses msg file format"""
+    format_name = 'msg'
+
+    def __init__(self, target_account, **kwargs):
+        print("Parsity parse")
+        # code goes here to set up mailbox and pull out any relevant account_data
+        account_data = {}
+        messages = []
+
+        self.file = target_account.input_path
+        print("Reading :", self.file)
+
+    def account_data(self):
+        return account_data
+
+    def messages(self):
+        for filename in glob.glob(self.file, recursive=True):
+            mail = extract_msg.openMsg(filename)
+            
+            message = {
```

Comment:
This is good as far as it goes, but since we're recursively globbing, we're going to need to track and add path information here - nesting on the filesystem is going to be how directories of msg files represent the info carried by Mail_Folder in mbox format, if I'm correct on this? @gwiedeman?

Reply:
Yes, I think so. I haven't tested the msg parser yet. I'm thinking of the parsers as just minimally-working examples for now, so I'm fine with merging this once its fixed, but keeping the different parser issues open and getting them fully working in the next iteration.